misc/windows-gaming.sh

```diff
@@ -0,0 +1,24 @@
+#!/bin/sh
+
+# This needs to be located in /usr/lib/systemd/system-sleep/
```

Comment:
Quoting `systemd-suspend(8)`:

>Note that scripts or binaries dropped in /usr/lib/systemd/system-sleep/ are intended for local use only and should be considered hacks. If applications want to be notified of system suspend/hibernation and resume, there are much nicer interfaces available.

As I explained in the issue, a sleep inhibitor is pretty definitely what we want here.